.github/workflows/CI.yml

```diff
@@ -0,0 +1,43 @@
+name: wok-CI
+on: ["push"]
+
+jobs:
+  build-and-test-wok:
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+
+    runs-on: ${{ matrix.os }}
+    steps:
+      - name: checkout
+        uses: actions/checkout@v1
+
+      - name: install pyyaml
+        run: |
+          sudo apt install python3-setuptools python3-dev
+          pip3 install pyyaml 
+
+      - name: update apt
+        run: |
+          sudo apt update 
+
+      - name: install distro depedencies
+        run: |
+          sudo apt install -y $(python3 -c 'import yaml;print(" ".join(yaml.load(open("dependencies.yaml"), Loader=yaml.FullLoader)["runtime-deps"]["ubuntu"]).replace("python3-cheetah", ""))')
+          sudo apt install -y $(python3 -c 'import yaml;print(" ".join(yaml.load(open("dependencies.yaml"), Loader=yaml.FullLoader)["development-deps"]["ubuntu"]))')
+      
```

Comment:
Also there are 2 files: ubuntu-dev-deps.list and ubuntu-runtime-deps.list which contain the parsed list of dependencies. If you want to use them instead of parsing the yaml file you can do `grep -v \# ubuntu-dev-deps.list` to remove the commented lines in the files.

Reply:
I tried to use them, but IDK why they were not visible

Reply:
https://github.com/ramonmedeiros/wok/commit/6cabce2b1e0dec088d169c7e988c3f009d9c86a9/checks?check_suite_id=347217929

```
grep: ubuntu-runtime-deps.list: No such file or directory
```

Reply:
You need to build the project to get them run `autogen + make` and they will be generated

Reply:
oh, but you need the deps before building the project, right?

Reply:
idk, what are the deps to run autogen e make?